x11-misc/polybar/polybar-3.4.3.ebuild

```diff
@@ -0,0 +1,66 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+XPP_COMMIT="044e69d05db7f89339bda1ccd1efe0263b01c8f6"
+I3IPCPP_COMMIT="cb008b30fc5f3febfe467884cb0211ee3c16386b"
+inherit cmake-utils python-single-r1
+
+DESCRIPTION="A fast and easy-to-use tool for creating status bars"
+HOMEPAGE="https://github.com/polybar/polybar"
+SRC_URI="https://github.com/polybar/polybar/releases/download/${PV}/${P}.tar"
+S="${WORKDIR}/${PN}"
+
+KEYWORDS="~amd64 ~x86"
+LICENSE="MIT"
+SLOT="0"
+
+IUSE="alsa curl i3wm ipc mpd network pulseaudio"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+DEPEND="
+	${PYTHON_DEPS}
+	$(python_gen_cond_dep 'x11-base/xcb-proto[${PYTHON_MULTI_USEDEP}]')
+	x11-libs/cairo[X,xcb(+)]
+	x11-libs/libxcb[xkb]
+	x11-libs/xcb-util-image
+	x11-libs/xcb-util-wm
+	x11-libs/xcb-util-xrm
+	alsa? ( media-libs/alsa-lib )
+	curl? ( net-misc/curl )
+	i3wm? (
+		dev-libs/jsoncpp
+		|| ( x11-wm/i3 x11-wm/i3-gaps )
+	)
+	mpd? ( media-libs/libmpdclient )
+	network? ( net-wireless/wireless-tools )
+	pulseaudio? ( media-sound/pulseaudio )
+"
+
```

Comment:
BDEPEND for EAPI 7? See [here](https://github.com/polybar/polybar/wiki/Compiling#build-dependencies) for a list of upstream's deps per category.

Reply:
I'm doing version bump, this should be for maintainer, so I'm not going to worry about the comments?